modules/openy_features/openy_demo_content/modules/openy_demo_bfooter/config/install/migrate_plus.migration.block_content_footer.yml

```diff
@@ -18,7 +18,7 @@ source:
         <ul class="list-inline">
         	<li><a class="fa fa-facebook" href="#">Facebook</a></li>
         	<li><a class="fa fa-twitter" href="#">Twitter</a></li>
-        	<li><a class="fa fa-youtube href="#">Youtube</a></li>
+        	<li><a class="fa fa-youtube" href="#">Youtube</a></li>
```

Comment:
This is good catch, however we still have to add `title=""` for all those links.